Jamfile

```diff
@@ -788,13 +788,24 @@ lib torrent
 
 headers = [ path.glob-tree include/libtorrent : *.hpp ] ;
 
-package.install install
+package.install install-torrent-lib
 	: <install-source-root>libtorrent
 	:
 	: torrent
 	: $(headers)
 	;
 
+local paths = [ package.paths "torrent-rasterbar" : ] ;
```

Comment:
the `package.paths` function doesn't take a string, it takes a list of requirements. as far as I can tell, the only requirement that it looks at is `<install-default-prefix>`, and if it isn't there, it returns the default prefix, which is platform specific.